### ISSUE TITLE
Can't get this to work with Heroku and MongoLab

### ISSUE THREAD

**Issue by gateway** [ParsePlatform/parse-server-example#2]
The server is up and running as well as a mongolab instance however their is no clear indication as to what you do next.  In the index.js their are several fields that look like they need to be filled out and Its not clear where you get the db connection and any app id or keys..

I used replaced the mongolab URI from my deployed system (im guessing thats what I needed to do) but where do you get myAppId and MasterKey.. If you say inside of parses admin I did grab those but I dont think thats right.. where can I find these keys or set them?

`var api = new ParseServer({
  databaseURI: process.env.DATABASE_URI || 'mongodb://localhost:27017/dev',
  cloud: process.env.CLOUD_CODE_MAIN || __dirname + '/cloud/main.js',
  appId: 'myAppId',
  masterKey: 'myMasterKey'
});`

Could you please be more descriptive on how to do this step by step because I'm sure lots of people will be wanting to know.

**Comment by jamesyu** [comment]
Take a look at this step by step instructions: https://parse.com/docs/server/guide#deploying

For myAppId and MasterKey, you can specify anything you want. If you're migrating, you'll want to set those to what they were from your hosted Parse app. Check out the migration guide here: https://parse.com/docs/server/guide#migrating

**Comment by gateway** [comment]
Yea I did, still not 100% clear, you or who ever wrote it , wrote it from a stand point if you know or have done it a few times.  There should be more of nicer newbie how to guide.  I followed all instructions for Heroku and MongoLab using the startup git depo here.  I did a test curl post to the server and all I got was cannot post returned.. so obviously I'm missing something and that would lead to documentation.  I know it might seem trivial to someone who has done this but I would ask that someone takes the time to really do more detailed on.  If I can get it up and running I dont mind but I'm a bit stuck since the depo seems more coded towards local than the other hosted option.. Im sure you understand and with massive announcement and struggle for people it would be in parse's best interest to provide more detailed docs.. thanks..
